install/install.ps1

```diff
@@ -8,87 +8,196 @@ $script:ErrorActionPreference='Stop'
+    if ($currentPath -NotMatch 'AzureAuth') {
+        Write-Verbose "Updating `$PATH to include ${latestDirectory}"
+        $newPath = if ($currentPath -Match $null) {
+            "${latestDirectory}"
+        } else {
+            "${currentPath};${latestDirectory}"
+        }
+        Set-ItemProperty -Path $registryPath -Name PATH -Value $newPath
+    }
+
+    Write-Output "Installed azureauth $version!"
 
-# We use a directory junction here because not all Windows users will have permissions to create a symlink.
-# We create this junction with cmd.exe's mklink because it has a stable interface across all active versions of Windows and Windows Server,
-# while PowerShell's New-Item has breaking changes and doesn't have the -Target param in 4.0 (the default PowerShell on Win Server 2012).
 
-Write-Verbose "Linking ${latestDirectory} to ${extractedDirectory}"
-cmd.exe /Q /C "mklink /J `"$latestDirectory`" `"$extractedDirectory`"" > $null
-if (!$?) {
-    Write-Error "Linking failed!"
 }
 
-Write-Verbose "Removing ${zipFile}"
-Remove-Item -Force $zipFile
-
-# Permanently add the latest directory to the current user's $PATH (if it's not already there).
-# Note that this will only take effect when a new terminal is started.
-$registryPath = 'Registry::HKEY_CURRENT_USER\Environment'
-$currentPath = (Get-ItemProperty -Path $registryPath -Name PATH -ErrorAction SilentlyContinue).Path
-if ($currentPath -NotMatch 'AzureAuth') {
-    Write-Verbose "Updating `$PATH to include ${latestDirectory}"
-    $newPath = if ($currentPath -Match $null) {
-        "${latestDirectory}"
+function V2-No-Latest{
+    param()
+
+    Write-Output "Using the Install No Latest Version"
+
+    $repo = if ([string]::IsNullOrEmpty($Env:AZUREAUTH_REPO)) { 'AzureAD/microsoft-authentication-cli' } else { $Env:AZUREAUTH_REPO }
+    $releaseName = "azureauth-${version}-win10-x64"
+    $releaseFile = "${releaseName}.zip"
+    $releaseUrl = "https://github.com/${repo}/releases/download/${version}/$releaseFile"
+
+    $azureauthDirectory = if ([string]::IsNullOrEmpty($Env:AZUREAUTH_INSTALL_DIRECTORY)) {
+        ([System.IO.Path]::Combine($Env:LOCALAPPDATA, "Programs", "AzureAuth"))
     } else {
-        "${currentPath};${latestDirectory}"
+        $Env:AZUREAUTH_INSTALL_DIRECTORY
+    }
+    $extractedDirectory = ([System.IO.Path]::Combine($azureauthDirectory, $releaseName))
+#    $latestDirectory = ([System.IO.Path]::Combine($azureauthDirectory, "latest"))
+    $versionPath = ([System.IO.Path]::Combine($azureauthDirectory, $version))
+    $zipFile = ([System.IO.Path]::Combine($azureauthDirectory, $releaseFile))
+
+    Write-Verbose "Creating ${azureauthDirectory}"
+    $null = New-Item -ItemType Directory -Force -Path $azureauthDirectory
+
+#   TODO uncomment downloading
+#    Write-Verbose "Downloading ${releaseUrl} to ${zipFile}"
+#    $client = New-Object System.Net.WebClient
+#    $client.DownloadFile($releaseUrl, $zipFile)

```

Comment:
Ah, I see. You're just commenting it out now to avoid doing network requests every time you run and test?